test/lib/jdk/test/lib/security/OpensslArtifactFetcher.java

```diff
@@ -48,43 +49,40 @@ public class OpensslArtifactFetcher {
            and return that path, if download fails then return null.
      *
      * @return openssl binary path of the current version
-     * @throws SkippedException if a valid version of OpenSSL cannot be found
+     * @throws SkippedException if OpenSSL is not available on the target platform
```

Comment:
Since `SkippedException` is now thrown in all cases, the JavaDoc `@throws` clauses should be merged.
```suggestion
     * @throws SkippedException if a valid version of OpenSSL cannot be found
     *         or if OpenSSL is not available on the target platform
```